docs/build/macos.md

```diff
@@ -48,73 +56,83 @@ These dependencies are also listed below if you'd prefer to install manually.
-  pyenv install 2.7.18
-  pyenv global 2.7.18
-  echo 'eval "$(pyenv init --path)"' >> ~/.bash_profile
-  ```
+```shell
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+brew install bash jq python3 meson ninja coreutils pandoc pyenv
+# Python 2.7 required for DeviceKit (among others?) at present
+pyenv install 2.7.18
+pyenv global 2.7.18
+echo 'eval "$(pyenv init --path)"' >> ~/.bash_profile
+```
 
-  On macOS, you will need to adjust your PATH so that coreutils’ `realpath` takes precedence over the BSD one:
+On macOS, you will need to adjust your PATH so that coreutils’ `realpath` takes
+precedence over the BSD one:
 
-  ```shell
-  # Credit: brew info coreutils
-  PATH="$HOMEBREW_PREFIX/opt/coreutils/libexec/gnubin:$PATH"
-  ```
+```shell
+# Credit: brew info coreutils
+PATH="$HOMEBREW_PREFIX/opt/coreutils/libexec/gnubin:$PATH"
+```
 
-* Web: node.js 18+, emscripten, openjdk 8
+## KeymanWeb Dependencies
 
-  ```shell
-  brew install node emscripten openjdk@8
-  ```
+* node.js 18+, emscripten, openjdk 8
 
-  Note: if you install emscripten with brew on macOS, only emscripten binaries
-  are added to the path via symlinks. This makes it reasonably safe to have
-  emscripten on the path, unlike on other platforms where emscripten also ends
-  up adding its versions of node, python, and other binaries to the path.
+```shell
+brew install node emscripten openjdk@8
+```
 
-* iOS: swiftlint, carthage
+Note: if you install emscripten with brew on macOS, only emscripten binaries are
+added to the path via symlinks. This makes it reasonably safe to have emscripten
+on the path, unlike on other platforms where emscripten also ends up adding its
+versions of node, python, and other binaries to the path.
 
-  ```shell
-  brew install swiftlint carthage
-  ```
+## Keyman for iOS Dependencies
 
-* macOS: carthage, cocoapods
+* XCode, swiftlint, carthage
 
-  ```shell
-  brew install carthage cocoapods
-  ```
+```shell
+brew install swiftlint carthage
+```
 
-* Android: openjdk 8, Android SDK, Android Studio, Ant, Gradle, Maven
+## Keyman for Mac Dependencies
 
-  ```shell
-  brew install openjdk@8 android-sdk android-studio ant gradle maven
-  # update path
-  source ../resources/devbox/macos/keyman.macos.env.sh
-  # optionally install sdk images
-  sdkmanager "system-images;android-30;google_apis;armeabi-v7a"
-  sdkmanager --update
-  sdkmanager --licenses
-  ```
+* XCode, carthage, cocoapods
 
-* kmc (optional): node
-  - Required to build keyboards using kmc
+```shell
+brew install carthage cocoapods
+```
 
-  ```bash
-  brew tap homebrew/cask-versions
-  brew install --cask --no-quarantine wine-stable
-  ```
+## Keyman for Android Dependencies
 
-* sentry-cli (optional)
-  - Uploading symbols for Sentry-based error reporting
+* openjdk 8, Android SDK, Android Studio, Ant, Gradle, Maven
 
-  ```
+```shell
+brew install openjdk@8 android-sdk android-studio ant gradle maven
```

Comment:
Don't we use openjdk@11 on master? (l.109 too)

Reply:
```suggestion
brew install openjdk@11 android-sdk android-studio ant gradle maven
```